code/datums/diseases/advance/symptoms/confusion.dm

```diff
@@ -53,9 +53,8 @@ Bonus
 				to_chat(M, "<span class='warning'>[pick("Your head hurts.", "Your mind blanks for a moment.")]</span>")
 		else
 			to_chat(M, "<span class='userdanger'>You can't think straight!</span>")
-			M.confused = min(100 * power, M.confused + 8)
+			M.confused = min(100, M.confused + (16 * power))

```

Comment:
Could potentially reduce confused.

Reply:
?

Reply:
If confused is above 100 from other sources, this would decrease it.

Reply:
That's how the symptom worked before, so I was hesitant to change it.

I don't think that it will matter too much, seeing as how you walk in completely random directions whenever you try to move at >40 confusion. 100 is a pretty high cap, is what I'm saying.

Reply:
This needs to be changed to something like

```DM
if(M.confused < 100)
    M.confused += 16*power
```

If you want caps, this is not the way to implement them.

Reply:
@ExcessiveUseOfCobblestone What if I used a CLAMP? Would that be acceptable?

Also, how exactly am I supposed to edit a closed PR? Am I supposed to ignore that greyed out edit button and go back to my virologypatch branch and edit it? Will the edits show up here as they used to do before?